iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VoltageAngleLimit.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+import java.util.Optional;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public interface VoltageAngleLimit extends OperationalLimits {
+
+    @Override
+    default LimitType getLimitType() {
+        return LimitType.VOLTAGE_ANGLE;
+    }
+
+    /**
+     * Return the mandatory name.
+     */
+    String getName();
```

Comment:
Even if not an `Identifiable` we should rename it ID if we want to ensure unicity

Reply:
I think it is name because it is name in cgmes, do you want it to be renamed id anyway ?